components/plugin/src/miniboxing/plugin/metadata/MiniboxMetadataAddons.scala

```diff
@@ -49,6 +49,12 @@ trait MiniboxMetadataAddons {
       sym hasAnnotation GenericClass
     }
     def isField = sym.isValue && !sym.isMethod
+    def isMbArrayMethod: Boolean = isSymbolMbArrayMethod(sym)
+    def isArray: Boolean = sym == ArrayClass
+    def isImplicitlyPredefMethod: Boolean = isPredefMemberNamed(sym, nme.implicitly)
+    def isCastSymbol: Boolean = definitions.isCastSymbol(sym)
+    def isIsInstanceOfAnyMethod: Boolean = sym == Any_isInstanceOf
+    def isArrowAssocMethod: Boolean = isPredefMemberNamed(sym, nme.Predef.newName("ArrowAssoc")) || (sym.owner == getMemberClass(PredefModule, TypeName("ArrowAssoc")) && sym.name == nme.MINGT)
```

Comment:
Cool! But why did you use `nme.Predef.newName`? You could have just used `newTermName`:
```
$ scala
Welcome to Scala version 2.11.5-20141107-114204-b1e6f57b2b (Java HotSpot(TM) 64-Bit Server VM, Java 1.7.0_80).
Type in expressions to have them evaluated.
Type :help for more information.

scala> :power
** Power User mode enabled - BEEP WHIR GYVE **
** :phase has been set to 'typer'.          **
** scala.tools.nsc._ has been imported      **
** global._, definitions._ also imported    **
** Try  :help, :vals, power.<tab>           **

scala> nme.Predef.newName("ArrowAssoc")
res0: $r.intp.global.TermName = ArrowAssoc

scala> newTermName("ArrowAssoc")
res1: $r.intp.global.TermName = ArrowAssoc

scala> res0 == res1
res2: Boolean = true
```